src/ui/projects/CreateProjectPage.js

```diff
@@ -122,6 +122,9 @@ export default function CreateProjectPage({ history, location }) {
                   <SearchInput placeholder="Search scenes..." value={params.q} onChange={onChangeQuery} />
                 </ProjectGridHeaderRow>
                 <ProjectGridHeaderRow>
+                  <Button as={Link} to="/scenes/new">
+                    Upload GLB
```

Comment:
I think we need to change the language of this button. Maybe "Import Project"? I think we may want to leave "GLB" out of it.

Reply:
Hmm, I think "project" would be confusing, since its not actually a spoke project you are importing (or creating). Its obviously weird since its in the projects flow and surfaced on the projects page right now, but I do think mentioning GLB seems important. Maybe something like "create project from GLB"?